### ISSUE TITLE
Android app hangs (infinite loop)

### ISSUE THREAD

**Issue by mikejpeters** [getsentry/sentry-cordova#41]
When I run my cordova app on Android with sentry-cordovda 0.8.4 it hangs. When I run `adb logcat` I see the following being continuously logged in an infinite loop:

```
[INFO:CONSOLE(2)] "not implemented", source: file:///android_asset/www/plugins/sentry-cordova/dist/js/sentry-cordova.bundle.min.js
```

Adding the `console.error` in 82530d2 seems to have caused this. Here is what I think is happening:

1. `addBreadcrumb()` is called
2. Android plugin throws "not implemented"
3. `BrowserBackend` has no `addBreadcrumb` method so it can't fallback to browser implementation (?)
4. `console.error` is called, but it's monkey-patched by `raven-js` and ends up calling `addBreadcrumb` again somehow

**Comment by HazAT** [comment]
Hey, sorry for this stupid bug.
I released `0.8.5` quickly to remove the `console.error` but will come up with a better solution for this.

**Comment by HazAT** [comment]
This should be fixed in the latest version.
